src/content/docs/crowdin/project-management/creating-project.mdx

```diff
@@ -28,15 +29,48 @@ When creating a project, you can select the preferred *project visibility*:
 
-In string-based projects, the focus is on managing translatable content as individual strings rather than source files. In this project type, after uploading source files, Crowdin parses them into source strings. Unlike file-based projects, source files aren't stored, and the emphasis is on managing the content at the string level. This approach is beneficial when dealing with projects that involve continuous content updates, dynamic content, or where a string-oriented structure is preferred.
+Crowdin supports [a wide range of formats](/supported-formats/) used in software development and documentation, including Android XML, iOS Strings, JSON, XLIFF, Markdown, DOCX, and more.
+
+#### Use Cases
+
+* Translate app strings and documents stored in structured files.
+* Maintain the original folder and file structure when exporting translations.
+* Keep source files in sync with code repositories.
+* Align localization with development and release workflows.
+* Suitable for version-controlled content or structured documents.
+
+### <Icon name="mdi:format-list-bulleted" class="inline-icon" />String-based Project
+
+A string-based project in Crowdin is designed for managing dynamic content through a centralized string repository. Instead of storing source files, Crowdin extracts individual translatable strings and maintains them within the platform. This setup is ideal for content that changes frequently, such as UI text and other modular content blocks.
+
+With this approach, you can update source strings directly in Crowdin without needing to re-upload files. Translated strings can then be exported in various formats, including JSON, iOS Strings, or Android XML, facilitating reuse across multiple platforms like web, mobile, and desktop.
+
+<Aside type="caution" title="Limitations">
+  Some connectors have limited support, and certain features may not be available in string-based projects.
+</Aside>
+
+#### Use Cases
+
+* Manage and translate app or website UI text in a centralized location.
+* Let content writers update source strings online without developer involvement.
+* Translate database content without needing to manage source files.
+* Avoid duplicate translation effort for strings reused across platforms (e.g., Android, iOS, Web).
+* Export translations in your preferred format, including JSON, iOS Strings, or Android XML.
+* Ideal for teams working with dynamic content and frequent updates.
+
+<QuestionAnswer title="How do I choose between a File-based and String-based project?">
+  Use a **File-based** project if you work with structured files and need to preserve the original structure and format.
+
+  Choose a **String-based** project if your content is frequently updated, reused across platforms, or managed outside of source files (e.g., in internal tools or databases).
```

Comment:
I would suggest to remove this `Question Answer` and extend a bit the respective step description below